### PR TITLE
Seed review fixes

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: src/app/components/about/about.ts [hunk 1]
```diff
@@ -5,14 +5,7 @@ import {Http} from 'angular2/http';
 @Component({
   selector: 'about',
   templateUrl: 'app/components/about/about.html',
-  styleUrls: ['app/components/about/about.css'],
-  providers: [],
-  directives: [],
-  pipes: []
```
Comment (Foxandxss) [inline src/app/components/about/about.ts]: This project is show a show off of what angular2 does, so this stuff is just for demonstration purposes.

Comment (gdi2290) [inline src/app/components/about/about.ts]: I would definitely keep them

**Comment by Foxandxss** [comment]
This can be closed too imo

**Comment by pkozlowski-opensource** [comment]
@Foxandxss huh? There are 3 commits in there

**Comment by Foxandxss** [comment]
I see. All the stuff you wanted to remove is just as a demo display.

**Comment by pkozlowski-opensource** [comment]
will drop the last commit

**Pull request closed without merging**